src/test/java/com/salesforce/mirus/config/SourceConfigTest.java

```diff
@@ -9,20 +9,27 @@
 package com.salesforce.mirus.config;
 
 import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.Matchers.contains;
 import static org.junit.Assert.assertThat;
 
+import java.util.Collections;
 import java.util.HashMap;
+import java.util.List;
 import java.util.Map;
+import org.apache.kafka.common.config.ConfigDef;
+import org.apache.kafka.connect.connector.ConnectRecord;
+import org.apache.kafka.connect.transforms.Transformation;
 import org.junit.Before;
 import org.junit.Test;
 
-public class SourceConfigTest {
+public class SourceConfigTest<R extends ConnectRecord<R>> {
```

Comment:
The type parameter on the test is a little weird. Is there a better way?

Reply:
Why not just make `SimpleTransformation` a simple non-nested class? Can probably still be in the same file if you make it package-private instead of public.